dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -112,3 +112,6 @@ sameSiteCookieStrict=Set the SSO cookie SameSite attribute value to Strict
 
 loggedOutCookieCacheCacheRef=JCache cache reference
 loggedOutCookieCacheCacheRef.desc=The JCache cache reference that is used as the logged-out-cookie cache.
+
+useContextRootAsCookiePath=Use context root as cookie path
+useContextRootAsCookiePath.desc=Specifies that the cookie path equals the context root of the web module instead of /
```

Comment:
is the end of this message missing or is it pulled in from elsewhere?

Reply:
no.

Reply:
The sentence is not complete, it's missing a direct object. The context root is used as the cookie path instead of what?

Reply:
instead of back slash